packages/components/src/mobile/bottom-sheet/stepper-cell/index.native.js

```diff
@@ -0,0 +1,162 @@
+const DEFAULT_STEP = 1;
+
+class BottomSheetStepperCell extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.announceValue = this.announceValue.bind( this );
+		this.onDecrementValue = this.onDecrementValue.bind( this );
+		this.onDecrementValuePressIn = this.onDecrementValuePressIn.bind( this );
+		this.onIncrementValue = this.onIncrementValue.bind( this );
+		this.onIncrementValuePressIn = this.onIncrementValuePressIn.bind( this );
+		this.onPressOut = this.onPressOut.bind( this );
+		this.startPressInterval = this.startPressInterval.bind( this );
+	}
+
+	componentWillUnmount() {
+		clearTimeout( this.timeout );
+		clearInterval( this.interval );
+		clearTimeout( this.timeoutAnnounceValue );
+	}
+
+	onIncrementValue() {
+		const { step, maxValue, onChangeValue, value } = this.props;
+		const newValue = value + step;
+
+		if ( newValue <= maxValue ) {
+			onChangeValue( newValue );
+			this.announceValue( newValue );
+		}
+	}
+
+	onDecrementValue() {
+		const { step, minValue, onChangeValue, value } = this.props;
+		const newValue = value - step;
+
+		if ( newValue >= minValue ) {
+			onChangeValue( newValue );
+			this.announceValue( newValue );
+		}
+	}
+
+	onIncrementValuePressIn() {
+		this.onIncrementValue();
+		this.timeout = setTimeout( () => {
+			this.startPressInterval( this.onIncrementValue );
+		}, 500 );
+	}
+
+	onDecrementValuePressIn() {
+		this.onDecrementValue();
+		this.timeout = setTimeout( () => {
+			this.startPressInterval( this.onDecrementValue );
+		}, 500 );
+	}
+
+	onPressOut() {
+		clearTimeout( this.timeout );
+		clearInterval( this.interval );
+	}
+
+	startPressInterval( callback ) {
+		let counter = 0;
+		this.interval = setInterval( () => {
+			callback();
+			counter += 1;
+
+			if ( counter === 10 ) {
+				clearInterval( this.interval );
+				this.startPressInterval( callback, STEP_SPEED / 2 );
+			}
+		}, STEP_SPEED );
+	}
+
+	announceValue( value ) {
+		const { label } = this.props;
+
+		if ( Platform.OS === 'ios' ) { // On Android it triggers the accessibilityLabel with the value change
+			clearTimeout( this.timeoutAnnounceValue );
+			this.timeoutAnnounceValue = setTimeout( () => {
+				AccessibilityInfo.announceForAccessibility( `${ value } ${ label }` );
+			}, 300 );
+		}
+	}
+
+	render() {
+		const { label, icon, minValue, maxValue, value, separatorType } = this.props;
+		const isMinValue = value === minValue;
+		const isMaxValue = value === maxValue;
+
+		const accessibilityLabel = sprintf(
+			/* translators: accessibility text. Inform about current value. %1$s: Control label %2$s: Current value. */
+			_x( '%1$s. Current value is %2$s', 'Increase or decrement to change the value' ),
+			label, value
+		);
+
+		return (
+			<View
+				accessible={ true }
+				accessibilityRole="adjustable"
+				accessibilityLabel={ accessibilityLabel }
+				accessibilityActions={ [
+					{ name: 'increment', label: 'increment' },
+					{ name: 'decrement', label: 'decrement' },
+				] }
```

Comment:
From what I see [here](https://facebook.github.io/react-native/docs/accessibility#accessibility-actions), labels for standard actions (like increment/decrement) are not required, but if we add them, the best would be to localise them. I'd suggest to not add them in this case and let the system do their magic.

Reply:
Sounds good! Way better to avoid handling those localizations =D